docs/src/builtins/steel_strings.md

```diff
@@ -1,8 +1,11 @@
 # steel/strings
+
 #### steel/strings
 
-Strings in Steel are immutable, fixed length arrays of characters. They are heap allocated,
-and are implemented under the hood as referenced counted rust `Strings`.
+Strings in Steel are immutable, fixed length arrays of characters. They are heap
+allocated, and are implemented under the hood as referenced counted Rust
+`Strings`. Rust `Strings` are stored as UTF-8 encoded bytes.
+
```

Comment:
Thanks for these doc changes! These builtin docs get generated from the doc comments in the rust code, can you also dupe the changes over there?

See here for example: https://github.com/mattwparas/steel/blob/4cf5e15d415c0d0d17b22e40e6c5551fb117962b/crates/steel-core/src/primitives/strings.rs#L17

Same goes for the `string-length` function.

I have a branch going parallel to this that will help in the future, you'll just need to run `cargo xtask docgen` and it'll populate the builtin docs, but for now just duping the changes over should suffice

Reply:
Done.